### PR TITLE
Remove Elasticsearch code

### LINKED ISSUE #5524: Remove ElasticSearch code
Once upon a time we had an ElasticSearch add-on, however, we don't have it for a while.

Before you tackling this issue, check with me.
I'm waiting to see if someone will raise an objection in [here](https://groups.google.com/forum/#!topic/mozilla.tools.treeherder/-Dgulh0-Tng).

You can see some of the code that will need removing in [here](https://github.com/mozilla/treeherder/search?q=ElasticSearch&type=Code).

### PR DESCRIPTION
**Pull request by sh0aibsha1kh**
### Testing
I did a Ctrl+F for "elastic" and found no results
I ran `docker-compose run backend sh -c "./runtests.sh"` and I had no failures. These were my results:
582 passed, 33 skipped, 9 xfailed in 591.21 seconds

Close #5524

### CODE REVIEW

**Comment by sh0aibsha1kh** [comment]
It seems that travis CI is failing because of the two elasticsearch requirements I removed from requirements/common.txt. When I add them back, travis CI says it's invalid.

After doing some research online, this [link](https://github.com/pypa/pipenv/issues/1494) suggests that this might be something that needs to be fixed in master.

Any suggestions @armenzg ?

**Comment by sh0aibsha1kh** [comment]
Nevermind, I think I might have just made a mistake in my formatting. By copying the requirements/common.txt files from `master`, travis CI is now passing.

**Comment by armenzg** [comment]
Cool. I will look at this hopefully tomorrow. If not on Monday.
It's been a very busy week.

File: requirements/common.txt [hunk 1]
```diff
@@ -70,8 +70,8 @@ simplejson==3.16.0 \
     --hash=sha256:65b41a5cda006cfa7c66eabbcf96aa704a6be2a5856095b9e2fd8c293bad2b46 \
     --hash=sha256:c206f47cbf9f32b573c9885f0ec813d2622976cf5effcf7e472344bc2e020ac1
 
-newrelic==5.2.0.127 \
-    --hash=sha256:736c2544bccc2db5a015b1a0f2852ee855b43aec40e5439da54af68640ef3505
+newrelic==5.2.1.129 \
```
Comment (armenzg) [inline requirements/common.txt]: Please rebase your changes over `master` or `git merge master`. These should not show up in the diff of this PR.

File: tests/autoclassify/test_classify_failures.py [hunk 2]
```diff
@@ -50,7 +46,8 @@ def test_classify_test_failure(text_log_errors_failure_lines,
     for (error_line, failure_line), expected in zip(zip(*expected_classified),
                                                     classified_failures):
         assert list(error_line.classified_failures.values_list('id', flat=True)) == [expected.id]
-        assert list(failure_line.error.classified_failures.values_list('id', flat=True)) == [expected.id]
+        assert list(failure_line.error.classified_failures.values_list(
```
Comment (armenzg) [inline tests/autoclassify/test_classify_failures.py]: Why do I see these changes?
Is your IDE formatting the files it touches?

File: treeherder/autoclassify/matchers.py [hunk 3]
```diff
@@ -138,7 +63,7 @@ def crash_signature_matcher(text_log_error):
 
     if (failure_line.action != "crash" or
         failure_line.signature is None or
-        failure_line.signature == "None"):
+            failure_line.signature == "None"):
```
Comment (armenzg) [inline treeherder/autoclassify/matchers.py]: Could you please fix these extra white spaces?

File: tests/autoclassify/test_classify_failures.py (hunk 2 above)
Comment (sh0aibsha1kh) [reply tests/autoclassify/test_classify_failures.py]: Yes I have an autoformatter, I'll disable it.

**Comment by sh0aibsha1kh** [comment]
All comments have been addressed, I believe this PR is ready to merge. @armenzg

**Comment by armenzg** [comment]
Thank you @sh0aibsha1kh  for your hard work here. Congratulations on your first contribution to Mozilla!

Let me know if you have interests on specific projects or ways you would like to keep helping.

**Pull request merged**